docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qiskit_experiments.characterization import T2RamseyAnalysis\n",
```

Comment:
Add  sub title for this section, something like "Providing initial user estimates". Also would be nice to give 1-2 sentences about where initial estimates can come from

Reply:
Agree. Done.